model/ravelWrap.cc

```diff
@@ -439,7 +449,13 @@ namespace
   /// @throw if type does not match global dimension type
   void Ravel::setDimension(Dimension::Type type,const std::string& units)
   {
-    auto descr=description();
+    return setDimension(selectedHandle(), type, units);
```

Comment:
This compiles? I didn't think it possible to return a void. It would save a lot of contorted generic code if possible.
Apparently so: https://stackoverflow.com/questions/2249108/can-i-return-in-void-function

Just learned something new.

Reply:
I don't remember doing that very intentionally, but yeah, no complaints from the compiler. There are some other languages I know that allow this, even though it's always a word more verbose than necessary :)